microhapulator/pipeaux.py

```diff
@@ -34,6 +34,7 @@ def full_reference_index_files(fasta):
 
 
 def parse_flash_summary(logfile):
+    # FIXME delete once this refactor is through
     with open(logfile, "r") as fh:
         data = fh.read()
         tp = re.search(r"Total pairs:\s+(\d+)", data).group(1)
```

Comment:
This is also an obsolete function, but it's being used by the `aggregate_summary` function. So this comment is a reminder to take care of it.